index.html

```diff
@@ -305,13 +305,13 @@ <h4 class="panel-title">
             </tr>
           </tbody>
           <tbody ng-repeat="bufferline in bufferlines">
-            <tr class="bufferline">
+            <tr class="bufferline" ng-class="::{'repeated-prefix': bufferline.prefixtext==bufferlines[$index-1].prefixtext, 'repeated-time': (((60*+bufferline.shortTime.substr(0, bufferline.shortTime.indexOf(':')) + +bufferline.shortTime.substr(bufferline.shortTime.indexOf(':')+1)) - (60*+bufferlines[$index-1].shortTime.substr(0, bufferlines[$index-1].shortTime.indexOf(':')) + +bufferlines[$index-1].shortTime.substr(bufferlines[$index-1].shortTime.indexOf(':')+1)) + 1440) % 1440) < 3, 'highlight': bufferline.highlight}">
```

Comment:
This should be an attribute of the bufferline. It's impossible to follow the logic when crammed onto a single line like this